components/drawer/events.md

```diff
@@ -131,3 +133,71 @@ The `ExpandedChanged` event fires every time the component's state is changed -
 @[template](/_contentTemplates/common/general-info.md#event-callback-can-be-async)
 
 @[template](/_contentTemplates/common/issues-and-warnings.md#valuechanged-lambda-required)
+
+## OnItemRender
+
+The `OnItemRender` event fires when each item in the Drawer renders. It allows you to customize the appearance of an item. Note that the event does not work alongside a [`<Template>`]({%slug drawer-templates%}#template), as the template disables all built-in features of the Drawer.
```

Comment:
Probably not all features, but all item-related features.

Reply:
The statement is in reference to the second paragraph of the Template section in the Drawer Templates article:

> When using this template all built-in features of the Drawer are disabled and should be implemented by the application, for example, the item selection and rendering, navigation to different pages, etc.

Reply:
Well, the content in the Templates article can be polished too. For example, Drawer expand and collapse or mini mode are features, which are not related to templates.